receiver/hostmetricsreceiver/internal/scraper/processscraper/config.go

```diff
@@ -36,8 +38,62 @@ type Config struct {
 	// collector does not have permission for.
 	// See https://github.com/open-telemetry/opentelemetry-collector/issues/3004 for more information.
 	MuteProcessNameError bool `mapstructure:"mute_process_name_error,omitempty"`
+
+	// FilterConfig is used to filter the set of processes that are reported on
+	Filters []FilterConfig `mapstructure:"filters"`
```

Comment:
Please do not use this old style config, use the new language as defined in the "transformprocessor."

Reply:
@bogdandrutu you mean using TQL (Telemetry Query Language)? I think it's currently defined within the transformprocessor only and we should have move it to internal collector API first. Unless it's something else you refer to?

Having said that, if we need TQL first, should we hold on until it's available? There are some other filters defined in the hostmetricsreceiver already which would need to be migrated